src/use-swr.ts

```diff
@@ -141,29 +145,22 @@ export const useSWRHandler = <Data = any, Error = any>(
         delete CONCURRENT_PROMISES_TS[key]
       }
 
-      // start fetching
-      try {
-        cache.set(keyValidating, true)
-        setState({
-          isValidating: true
-        })
-        if (!shouldDedupe) {
```

Comment:
now it will broadcast anyway, will this results into more revalidations?

Reply:
Good catch!

Reply:
Fixed